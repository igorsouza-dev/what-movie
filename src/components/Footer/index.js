import React from 'react';

export default function Footer() {
  return <div>Film data provided by TMDB</div>;
}
