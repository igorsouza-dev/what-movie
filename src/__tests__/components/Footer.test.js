import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from 'components/Footer';

describe('<Footer />', () => {
  afterEach(cleanup);
  it('should render', () => {
    const { container, getByText } = render(<Footer />);

    const linkElement = getByText(/Data provided by/i);
    expect(linkElement).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
