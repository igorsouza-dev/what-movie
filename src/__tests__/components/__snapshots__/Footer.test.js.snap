// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<div>
  Film data provided by TMDB
</div>
`;
